modules/conversion-center/base/widget-link-in-bio-base.php

```diff
@@ -1755,50 +1688,20 @@ protected function add_identity_image_cover_control( array $condition ): void {
 			]
 		);
 
-		$this->add_control(
-			'identity_image_show_bottom_border',
+		$this->add_borders_control(
+			'identity_image_bottom',
```

Comment:
@willhowat did you check that this works?

Reply:
@nicoladj77 yes